src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js

```diff
@@ -1,32 +1,47 @@
-import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
+import React from 'react';
```

Comment:
file extension should be `.jsx` if using React

Reply:
changed!